app/models/spotlight/resource.rb

```diff
@@ -100,6 +102,19 @@ def touch_exhibit!
       def write?
         Spotlight::Engine.config.writable_index
       end
+
+      def cleanup_solr_record
+        blacklight_solr.delete_by_id(document_ids, params: { softCommit: true })
+      end
+
+      def document_ids
+        document_builder.documents_to_index.to_a.map { |y| y[:id] }
```

Comment:
Thanks for the suggestions. I did not know Solr documents could appear in multiple exhibits. But, that makes since to allow them to. I assume removal of the orphaned resource is a must when an exhibit is deleted. The Solr index would then have to be amended appropriately. I will look into making the Solr cleanup configurable and how to cleanup Solr entries without removing documents used within other exhibits. What do you think about the upload image cleanup upon destroy of the resource?